linux_example/trustx_verify.c

```diff
@@ -247,7 +247,12 @@ int main (int argc, char **argv)
 			}
 		}
     } while (0); // End of DO WHILE FALSE loop.
- 
+
+    // If -b argument is given but others are not then exit
+    if (optiga_oid == 0) {
```

Comment:
This check needs to be modified because after testing it was determined that trustx_verify can use a host pubkey only instead of reading the certificate from OID.

Reply:
fixed in latest commit: "fix if statement so that trustx_verify..."